src/app/tabs/runTab/settings.js

```diff
@@ -127,7 +127,7 @@ class SettingsUI {
     `
 
     const el = yo`
-      <div class="${css.settings}">
+      <div class="${css.settings} module-section">
```

Comment:
what is module-section ?

Reply:
This class necessary to style elements only in dark and themes

Reply:
why can't it be in all of them?